libs/confman/testing/CMakeLists.txt

```
@@ -3,4 +3,6 @@ if(BUILD_EXAMPLES)
 endif()
 add_subdirectory(mocks)
 add_subdirectory(stubs)
-add_subdirectory(tests)
+if(BUILD_TESTS)
```

```suggestion
if(BUILD_TESTING)
```
where is build_tests from?